traffic_monitor/cache/astats.go

```diff
@@ -38,7 +38,7 @@ import (
 	"github.com/apache/trafficcontrol/lib/go-log"
 	"github.com/apache/trafficcontrol/traffic_monitor/dsdata"
 	"github.com/apache/trafficcontrol/traffic_monitor/todata"
-	"github.com/json-iterator/go"
+	jsoniter "github.com/json-iterator/go"
```

Comment:
Why is this necessary?

Reply:
It wasn't, I didn't do it - at least not intentionally. I installed an IDE to help me wade through the confusing naming schemes of TM code, and I'd bet it did that for me automatically.

I can change it back if you want

Reply:
Ok, it doesn't seem to be affecting anything but it doesn't appear to be used either, which is weird because Go won't let you compile with unused imports. I guess it must be `init`ing something.